index.html

```diff
@@ -633,16 +633,31 @@ <h3>Multiple Schemas</h3>
         <section class="informative">
           <h3>Validity of a Verifiable Credential</h3>
           <p>
-            Validation against a [[JSON-SCHEMA]] may be confused with 
+            Validation against a [[JSON-SCHEMA]] may be confused with
             <a href="https://www.w3.org/TR/vc-data-model/#dfn-credential-validation">validation</a>
             or <a href="https://www.w3.org/TR/vc-data-model/#dfn-verify">verification</a>
             of a Verifiable Credential. A valid credential according to a [[JSON-SCHEMA]] refers
             only to the structure of the claims comprising a Verifiable Credential. This idea of
             validity does not imply anything about the validity of the Verifiable Credential itself.
-            It's possible for a Verifiable Credential to be considered valid by one verifier, 
+            It's possible for a Verifiable Credential to be considered valid by one verifier,
             while another verifier would not consider it valid.
           </p>
         </section>
+        <section class="informative">
+          <h3>Relationship to Verifiable Credential Type Property</h3>
+          <p>
+            A Verifiable Credential can have a specific <a href="https://www.w3.org/TR/vc-data-model/#dfn-type">type</a>
+            property. Implementers are encouraged to use the value of the specific type for the name property of
+            <a href="#metadata">metadata</a>.
+          </p>
+
+          <p>
+            It is important to note that a credential schema lets issuers communicate how to process the structure
+            of data inside a verifiable credential, whereas the VC type property lets issuers communicate the
+            semantics of the same data. It is advisable to have all properties that have a semantic definition
```

Comment:
>  It is important to note that a credential schema lets issuers communicate how to process the structure
            of data inside a verifiable credential, whereas the VC type property lets issuers communicate the
            semantics of the same data. It is advisable to have all properties that have a semantic definition
to be defined within the schema property of <a href="#metadata">metadata</a>.

Change to

```
 It is important to note that a <a>credential schema</a> enables issuers to communicate how to process the structure of data inside a verifiable credential, whereas the <code>type</code> property of a <a>verifiable credential</a>
lets issuers communicate the semantics of the data. It is RECOMMENDED to associate all properties that have a semantic mapping with a property in a <a>credential schema</a>.
```

I will note that the LD critics will say that all properties in a VC have a semantic meaning/mapping because LD is used as a data model